shared/build.gradle.kts

```diff
@@ -54,10 +56,21 @@ dependencies {
 	add("kspAndroid", libs.hilt.compiler)
 }
 
+multiplatformResources {
+	multiplatformResourcesPackage = "com.intive.picover.shared"
+	multiplatformResourcesClassName = "MR"
```

Comment:
`MR` is the default value and it sets how the resources are accessible from common code, in this case 
`MR.strings.UploadChannelName` 

@gmatyszczak What are your thoughts here? I think we could go with the default value. 
Using `R`, to follow android style could lead to same clashes and some strange and unexpected issue in the future. 

I've added this line explicitly just to discuss the possibilities. It will be removed, if we will decide to go with the default one.

Reply:
I agree, let's go with the default one